Userland/Libraries/LibWeb/CMakeLists.txt

```diff
@@ -296,6 +298,9 @@ function(libweb_js_wrapper class)
     add_custom_target(generate_${basename}Prototype.cpp DEPENDS Bindings/${class}Prototype.cpp)
 endfunction()
 
+libweb_js_wrapper(Communication/CloseEvent)
+libweb_js_wrapper(Communication/MessageEvent)
+libweb_js_wrapper(Communication/WebSocket)
```

Comment:
Looks like only `MessageEvent` is used for multiple things and should be under `Communication` (https://html.spec.whatwg.org/multipage/comms.html), can we move the other two to `WebSocket(s)`?